typing/types.mli

```diff
@@ -883,7 +883,7 @@ type label_description =
     lbl_mut: mutability;                (* Is this a mutable field? *)
     lbl_modalities: Mode.Modality.Value.Const.t;
                                         (* Modalities on the field *)
-    lbl_jkind : jkind_l;                (* Jkind of the argument *)
+    lbl_sort: Jkind_types.Sort.Const.t; (* Sort of the argument *)
```

Comment:
I'm a little worried whether, in the future, we might want to ensure record fields or constructor arguments are non-null or are external. But for now, it's not an issue.

Reply:
Yes! Very reasonable. This is part of why I think we've gotten the design of layouts and jkinds subtly wrong, because I think the details you mention (nullability and externality, specifically) might end up in the layout bit instead of the jkind bit. But we're not there in any case, and I'm pretty confident that `label_description` won't need a full jkinds (with mode-crossing behavior), so if we need externality/nullability, we'll just get it where it needs to be.

Reply:
I'm fine with this then.